AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/Agent.java

```diff
@@ -11,17 +11,17 @@
     private String firstName;
     private String lastName;
     private LocalDate dateOfBirth;
-    private int rank;
+    private int agentRank;
```

Comment:
Did this fix the problem?

Reply:
I think it was a combination of this and also where the user and password for the database is, mine was blank so i added back in 'agentdiscoveries' to both however didn't push that file.